snap/hooks/post-refresh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+if [ ! -d "${SNAP_DATA}/tempest_test_lists" ]; then
+  mkdir $SNAP_DATA/tempest_test_lists
+fi
+
+cp -r $SNAP/tempest_test_lists/* $SNAP_DATA/tempest_test_lists
```

Comment:
Is the `-r` needed? We are copying only files here.

If I understand the `post-refresh` is used when snap is refreshed? What do you say if we remove all lists and then copy new one? Because we can drop support for same lists, but they will be not removed if used refresh snap.

Reply:
Personally I would keep the `-r` flag in case in the future we want to organize the tests into sub-directories. And it doesn't hurt if we only have files.

For snap refreshes, if we want to remove files in the previous $SNAP_DATA directory, we need to do it in the `pre-refresh` hook, before the snap refreshed to a new revision. But I wonder whether it is necessary. Like versioned configuration, if we keep the files there, we could compare them across versions.

Reply:
Ok :+1:.

It could happen, that some list will stay there and tests in them will no longer exists. Or not?

Reply:
For future record, the reason for using `$SNAP_DATA` instead of `$SNAP_USER_DATA` is because we want to expose the built-in test lists for user to inspect but not to edit. And $SNAP_DATA doesn't exist in the snap, therefore requires customized `install` and `post-refresh` hooks.